src/tools/w3dviewer/camerasettingsdialog.cpp

```diff
@@ -0,0 +1,229 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D View camera distance dialog
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "camerasettingsdialog.h"
+#include "camera.h"
+#include "graphicview.h"
+#include "resource.h"
+#include "utils.h"
+#include "viewerscene.h"
+#include "w3dviewdoc.h"
+
+bool CameraSettingsDialogClass::m_isUpdating = false;
+
+// clang-format off
+BEGIN_MESSAGE_MAP(CameraSettingsDialogClass, CDialog)
+	ON_COMMAND(IDC_FOV, OnFieldOfView)
+	ON_COMMAND(IDC_CLIPPLANES, OnClipPlanes)
+	ON_COMMAND(IDABORT, OnReset)
+END_MESSAGE_MAP()
+// clang-format on
+
+CameraSettingsDialogClass::CameraSettingsDialogClass(CWnd *pParentWnd) : CDialog(IDD_CAMERA, pParentWnd) {}
```

Comment:
Can initialize `m_isUpdating` here.

Reply:
m_isUpdating is not a member variable and is initialized elsewhere.